src/Server.cpp

```diff
@@ -35,6 +38,36 @@ RouteCfg* find_route(const HttpRequest& req, std::vector<RouteCfg>& routes)
     return route_match.second;
 }
 
+/*
+*   @check_request_method:
+*    Checks if the requested method is accepted by the route
+*/
+bool    check_request_method(RouteCfg* route, const std::string method) {
+    for (size_t i = 0; i < route->accepted_methods.size(); i++)
+        if (method == route->accepted_methods[i])
+            return true;
+    return false;
+}
+
+/*
+*   @index_path:
+*       If auto_index is true, returns user request
+*       If not, return predefined index
+*
+*/
+std::string	index_path(const HttpRequest& req, RouteCfg* route)
+{
+    std::string	request = "." + route->root + route->route_path;
```

Comment:
Why is a `.` added in front of the root? Root should be an absolute path, not relative.

Reply:
Because the path bust be relative since the "/var/www" folder is not an absolute path